frontend/src/components/sidebar/sidebar.scss

```diff
@@ -0,0 +1,97 @@
+.sidebar-section {
+    min-height: 100%;
```

Comment:
Try to use bootstrap classes instead of custom styles.
Also, pull develop branch and use out default classes for color, font-size etc.